lldb/source/Expression/UserExpression.cpp

```diff
@@ -424,6 +425,14 @@ UserExpression::Execute(DiagnosticManager &diagnostic_manager,
                         const EvaluateExpressionOptions &options,
                         lldb::UserExpressionSP &shared_ptr_to_me,
                         lldb::ExpressionVariableSP &result_var) {
+  Debugger *debugger =
+      exe_ctx.GetTargetPtr() ? &exe_ctx.GetTargetPtr()->GetDebugger() : nullptr;
+  Progress progress("Running expression",
+                    m_options.IsForUtilityExpr()
+                        ? "(LLDB utility)"
```

Comment:
```suggestion
                        ? "LLDB utility"
```

Reply:
I think `Running expression: (LLDB utility)` would look weird in the terminal.

Reply:
I don't think utility functions ever get run through UserExpression::Execute, do they?  If you want to report utility functions, you need to send progress from `FunctionCaller::ExecuteFunction`.

Reply:
Also, for `UtilityFunction` specifically, it might be nice to show the name of the function being called as a progress detail.